egs2/babel/asr1/local/make_corpus_subset.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash 
+
+# Copyright 2012  Johns Hopkins University (Author: Yenda Trmal)
```

Comment:
Did you modify anything in this file?
If no modification, then use symbolic links to original recipe for `local` script as possible.
(I can't compare all, so just making sure)